src/Item/Item.service.spec.ts

```diff
@@ -98,6 +102,57 @@ describe('Item Service', () => {
         })
       })
     })
+
+    describe('when the collection db is published', () => {
+      describe('and the collection contains smart wearables', () => {
+        beforeEach(() => {
+          oldCreatorAddress = '0xoldCreator'
+          const dbCollection = {
+            ...dbCollectionMock,
+            eth_address: oldCreatorAddress,
+          }
+          dbItem = {
+            ...dbItemMock,
+            eth_address: oldCreatorAddress,
+            contents: {
+              ...dbItemMock.contents,
+              'male/game.js': 'hash1',
+              [VIDEO_PATH]: 'videoHash2',
+            },
+            video: 'videoHash',
+          }
+          ;(Item.findOne as jest.Mock).mockRestore()
+          ;(Item.findOne as jest.Mock).mockResolvedValueOnce(dbItem)
+          ;(Item.hasPublishedItems as jest.Mock).mockResolvedValue(true)
+          ;(Item.upsert as jest.Mock).mockImplementation((value) => value)
```

Comment:
Would you mind using the `once` version of the mocks? This will prevent the mocks from leaking into other tests.